src/ui/atoms/inputBase/InputBase.tsx

```diff
@@ -55,6 +55,10 @@ export type InputBaseProps = {
    * If true and multiline, the user can't resize the text area.
    */
   readonly disableAreaResize?: boolean
+  /**
+   * If true, the TextField will take 100% of its parent's size.
```

Comment:
If the `fullwidth` property is now part of the `InputBase` component, I will not mention `TextField` anymore in the property description as it can be used in any component that entends base props. 

```suggestion
   * If true, the input will take 100% of its parent's size.
```